_includes/head.html

```diff
@@ -11,6 +11,7 @@
 <title>{{ site.title | xml_escape }}{{ custom | xml_escape }}{% if title %} :: {{ title | xml_escape }}{% endif %}</title>
 {% if page.excerpt %}<meta name="description" content="{{ title | strip_html | xml_escape }}">{% endif %}
 <meta name="keywords" content="{{ page.tags | join: ', ' }}">
+<meta http-equiv="onion-location" content="http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion" />
```

Comment:
Also, does this result in redirections from other pages to / ?

Reply:
No, it redirects to the .onion homepage

Reply:
Wait, hmm? I mean if you go to bitcoincore.org/downloads, will it then redirect you to bitcoincore.onion/ instead of bitcoincore.onion/downloads?